betterproto/tests/inputs/import_service_input_message/request_argument.proto

```diff
@@ -0,0 +1,5 @@
+syntax = "proto3";
+
+message RequestArgument {
```

Comment:
What does this relate to?

Reply:
Thanks for reviewing 😄 
Accidental left-over. Will remove 👍